### PR TITLE
Tips and tricks

### LINKED ISSUE #194: Tips and Tricks Tutorial
@jamiehannaford inspired me!

to get the swarm discovery id for your cluster, you can run:

```
`docker inspect -f "{{index .Config.Cmd 6}}" $(docker ps -aq -f name=swarm-manager -n 1)` (edited)
```

Jamie Hannaford [5:26 AM]
@thomasem: managed to get a list of swarm nodes from that. just plug above cmd into `swarm list $(...)` (edited)

### PR DESCRIPTION
**Pull request by everett-toews**
Closes #194 

This is just the start. Add your tips and tricks in follow on PRs please!

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-0773d43f5b/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/getcarina.com/build-59525ff38a/).

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-59525ff38a/)

File: _reference/2016-03-30-tips-and-tricks.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+</div>
+
+{% for tip in page.tips %}
+  {% comment %}
+    This is a terrible, awful, no-good band-aid to parse one jekyll variable that we know gets used a lot in the frontmatter.
+  {% endcomment %}
+  {% assign baseUrlVar = '{{ site.baseurl }}' %}
+  <h4 id="{{ tip.title | slugify }}">{{ tip.title }}</h4>
+  <div class="answer">
+    {{ faq.body | replace: baseUrlVar, site.baseurl }}
```
Comment (ktbartholomew) [inline _reference/2016-03-30-tips-and-tricks.md]: @everett-toews this expression should begin with `tip.body` (not `faq.body` as it is now)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-960f7abb0c/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-f768b9e664/)

**Comment by carolynvs** [comment]
Will you be including `docker-ssh`, https://github.com/getcarina/getcarina.com/issues/194#issuecomment-203160689 or should I make an new PR for that once this ships?

**Comment by everett-toews** [comment]
@carolynvs Your own PR so I don't mess something up.

**Pull request merged**